packages/manager/apps/dedicated/client/app/account/user/users/translations/Messages_fr_FR.json

```
@@ -47,5 +47,72 @@
   "user_users_add_modal_description": "Description",
   "user_users_add_login_pattern_error": "Le login ne doit contenir que les caractères suivants : lettres, chiffres, ., - et _",
   "user_users_add_error_message": "Une erreur est survenue lors de l'ajout de l'utilisateur",
-  "user_users_add_success_message": "L'utilisateur <strong>{{login}}</strong> a été ajouté avec succès."
-}
\ No newline at end of file
+  "user_users_add_success_message": "L'utilisateur <strong>{{login}}</strong> a été ajouté avec succès.",
+  "user_users_sso_add": "Connexion SSO",
+  "user_users_sso_add_modal_title": "Se connecter en SSO",
+  "user_users_sso_add_modal_info": "Afin d'autoriser vos utilisateurs à utiliser votre propre identity provider, nous avons besoin de ses informations afin de configurer votre compte.",
+  "user_users_sso_add_modal_warning": "Attention : un identity provider externe désactivera les utilisateurs actuels.",
+  "user_users_sso_add_modal_step1_question": "Veuillez renseigner les détails du SSO",
+  "user_users_sso_add_modal_group_attribute_name": "Nom d'attribut de groupe (groupAttributeName)",
+  "user_users_sso_add_modal_xml_metadata": "Métadonnées XML (metadata)",
+  "user_users_sso_add_success_message": "La connexion SSO a été ajoutée avec succès.",
+  "user_users_sso_add_error_message": "Une erreur est survenue lors de l'ajout de la connexion SSO",
+  "user_users_sso_update_modal_title": "Modifier le SSO",
+  "user_users_sso_update_modal_info": "Afin d'autoriser vos utilisateurs à utiliser votre propre identity provider, nous avons besoin de ses informations afin de configurer votre compte.",
+  "user_users_sso_update_modal_step1_question": "Veuillez renseigner les détails du SSO",
+  "user_users_sso_update_modal_group_attribute_name": "Nom d'attribut de groupe (groupAttributeName)",
+  "user_users_sso_update_success_message": "La connexion SSO a été modifiée avec succès.",
+  "user_users_sso_update_error_message": "Une erreur est survenue lors de la mise à jour de la connexion SSO",
+  "user_users_sso_service_url": "SSO service URL",
+  "user_users_sso_creation_date": "Création",
+  "user_users_sso_last_update_date": "Dernière modification",
+  "user_users_sso_details": "Détails du SSO",
+  "user_users_sso_update": "Modifier le SSO",
+  "user_users_sso_delete": "Supprimer le SSO",
+  "user_users_sso_delete_modal_title": "Supprimer la connexion SSO",
+  "user_users_sso_delete_modal_step1_question": "Êtes-vous sur de vouloir supprimer la connexion SSO ?",
+  "user_users_sso_delete_success_message": "La connexion SSO a été supprimée avec succès.",
+  "user_users_sso_delete_error_message": "Une erreur est survenue lors de la suppression de la connexion SSO :",
```

Probably forget to display the error message.

```suggestion
  "user_users_sso_delete_error_message": "Une erreur est survenue lors de la suppression de la connexion SSO : {{errorMessage}}",
```

---

Error messages are managed in the controllers (design pattern followed in all the user-users part)